content/_code-samples/checks/py/cash_check.py

```diff
@@ -0,0 +1,75 @@
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.utils import xrp_to_drops
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+from xrpl.clients import JsonRpcClient
+from xrpl.models import CheckCreate, IssuedCurrencyAmount
+
+# create a client
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234")
+
+
+# cash an xrp check
+
+# check id
+check_id = str
+
+# amount to cash
+amount = 10.00
+
+# generate wallet
+sender_wallet = generate_faucet_wallet(client=client)
+
+# build check cash transaction
+check_txn = CheckCash(account=sender_wallet.classic_address, check_id=check_id, amount=xrp_to_drops(amount))
```

Comment:
CheckCash not imported, please run all scripts and make sure there are no errors, thanks!